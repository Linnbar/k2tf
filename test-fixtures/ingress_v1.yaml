apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: tls-example-ingress
spec:
  rules:
    - host: echo.example.com
      http:
        paths:
          - pathType: Prefix
            path: "/"
            backend:
              service:
                name: echo-server
                port:
                  number: 8080

          - pathType: Prefix
            path: "/.ws"
            backend:
              service:
                name: echo-server-http
                port:
                  name: http
  tls:
    - hosts:
        - echo.example.com