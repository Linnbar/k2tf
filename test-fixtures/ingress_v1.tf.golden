resource "kubernetes_ingress_v1" "tls_example_ingress" {
  metadata {
    name = "tls-example-ingress"
  }
  spec {
    tls {
      hosts = ["echo.example.com"]
    }
    rule {
      host = "echo.example.com"
      http {
        path {
          path      = "/"
          path_type = "Prefix"
          backend {
            service {
              name = "echo-server"
              port {
                number = 8080
              }
            }
          }
        }
        path {
          path      = "/.ws"
          path_type = "Prefix"
          backend {
            service {
              name = "echo-server-http"
              port {
                name = "http"
              }
            }
          }
        }
      }
    }
  }
}
